samples/nfc/system_off/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+set(NRF_SUPPORTED_BOARDS
+  nrf52_pca10040
+  nrf52840_pca10056
```

Comment:
PR with NFC support for nRF5340 (#1820) has just been merged to the master branch. Could you add nRF5340 target also to this sample and check if it works as expected?

Please also update Readme and yaml file for CI.